vacationtime-solution/README.md

```diff
@@ -0,0 +1,70 @@
+# Manage Employee Vacation Time Requests
+
+_Quickly approve/disapprove employees' vacation time requests, and 
+create calendar events or send notification emails accordingly.._
+
+Last updated: July, 2019
+
+A Google Form is a great tool for employees to quickly submit vacation
+time requests in a uniform manner. However, it can be difficult for employers
+to deal with everybody's requests at once, and send the correct e mails/create 
+the correct calendar events accordingly.
+
+This solution uses [Google Apps Script][apps-script] to generate a Google Sheet
+and apropriate columns based on employees' vacation time requests. When an employee
+submits a request through a Google Form, the employer solely needs to load the Form
+responses into a Google Sheet, run the Apps Script code, and mark approval/disapproval
+column values as necessary. The script will then notify employees of their disapproved
+requests through automated emails, or create a shared calendar event between
+manager and employee for those with approved requests.
+Now, managers can easily see on their calendars when certain employees will be on
+vacation, and have dealt with all other requests appropriately and with the simple
+click of a button.
+
+[apps-script]: https://developers.google.com/apps-script
+
+## Technology highlights
+
+- The script sends automated e mails to employees with 'NOT APPROVED' status
+  using [MailApp][mail-app].
+- The script creates shared calendar events between manager and employee using
+  [CalendarApp][cal-app].
+- The script sets certain columns to contain drop-down menu cells using 
+  [Data Validation rules][data-val].
+
+[mail-app]: https://developers.google.com/apps-script/reference/mail/mail-app
+[cal-app]: https://developers.google.com/apps-script/reference/calendar/calendar-app
+[data-val]: https://developers.google.com/apps-script/reference/spreadsheet/data-validation-builder
+
+
+## Try it out!
```

Comment:
Possible to simplify all this and make it more consistent with other solutions? The other sheet-based solutions start copying the sheet since that's an easy way to bundle things (the script gets copied too, so no copy/paste code.)

Copying from a sheet also simplifies the setup code since you can do a lot of the sheet formatting ahead of time in the sheet itself instead of via apps script. You'd still need some setup code to create/link forms and/or set up triggers thoguh.